.prettierrc.js

```diff
@@ -8,4 +9,22 @@ module.exports = {
     require("prettier-plugin-packagejson"),
     require("prettier-plugin-sh"),
   ],
+  overrides: [
+    {
+      files: [".eslintrc.cjs"],
+      options: {
+        quoteProps: "preserve",
+        semi: false,
+        trailingComma: "none",
+      },
+    },
+    {
+      files: ["blocks/**/.eslintrc.cjs"],
+      options: {
+        quoteProps: "as-needed",
+        semi: true,
+        trailingComma: "all",
+      },
+    },
+  ],
```

Comment:
Renaming `.json` to `.js` removes trailing commas, semicolons and quotes around keys by default:

```json
{
  "hello": "world"
}
```

```js
module.exports = {
  "hello": "world",
};
```

When too many lines have changed in a file, git marks JSON as deleted and CJS as created. This loses `git blame`. By keeping `.eslintrc.cjs` formatted as before helps git to mark most files as renamed.

I will revert to standard Prettier formats in one of the follow-up PRs. The continuity of `git blame` won't be an issue because file names will not change.